cpp/src/arrow/util/hashing.h

```diff
@@ -698,6 +808,23 @@ struct HashTraits<T, enable_if_fixed_size_binary<T>> {
   using MemoTableType = BinaryMemoTable;
 };
 
+template <typename MemoTableType>
+static inline Status ComputeValidityBitmap(MemoryPool* pool,
```

Comment:
We usually call it "null bitmap" ;-)

Reply:
"validity bitmap" is used in some other places (e.g. in the Arrow spec itself), it's my fault for using the "null bitmap" terminology in the first place in this codebase